test/utils/typed_value.hpp

```diff
@@ -0,0 +1,27 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include "types.hpp"
+#include "value.hpp"
+
+namespace fizzy::test
+{
+struct TypedValue
```

Comment:
Can you add some basic tests for this? I know... testing testing tools.

Reply:
Added.